README.md

```diff
@@ -9,29 +9,36 @@
 
 ### Setting up the project
 
-1. Copy the docker environment variables and fill in any missing secrets from the [secrets
-repository][secret-docker-compose]:
+1. Copy the docker environment variables:
 
 ```bash
-$ cp docker-compose.env.sample docker-compose.env
+cp docker-compose.env.sample docker-compose.env
```

Comment:
Together with this, I believe we need to do also something like
```
cp config/database.example.yml config/database.yml
```
which I forgot to add when I removed that file. It might still work without it as it reads the config from `DATABASE_URL` environment variable, but for it to work outside docker it's required.